climakitae/data_export.py

```diff
@@ -204,8 +205,8 @@ def _export_to_user(user_export_format,data_to_export,
         
     assert "xarray" in str(
```

Comment:
This should really be a `if..raise` instead of `assert` which is only really used in debugging.

Here is an example:

```
x = 10
if x > 5:
    raise Exception('x should not exceed 5. The value of x was: {}'.format(x))
```